modules/nw-create-custom-ingress-controller.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-create-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+As a cluster administrator, you can create a new custom Ingress Controller. Because the default Ingress Controller might be updated during {product-title} upgrades, creating a custom Ingress Controller can be helpful when maintaining a configuration manually that persists across cluster upgrades.
+
+This example provides a minimal spec for a custom Ingress Controller. To further customize your custom Ingress Controller, see "Configuring the Ingress Controller".
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a YAML file that defines the custom `IngressController` object:
++
+.Example `custom-ingress-controller.yaml` file
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+    name: <custom_name> <1>
+    namespace: openshift-ingress-operator
+spec:
+defaultCertificate: <2>
+    name: custom-ingress-custom-certs
+domain: <custom_domain> <3>
+----
+<1> Specify the a custom `name` for the `IngressController` object.
+<2> Set `spec.defaultCertificate` to the name of the secret with the custom wildcard certificate.
+<3> Specify the `domain` to your domain name. The domain specified on the `IngressController` object and the domain used for the certificate must match. For example, if the `domain` value is `"custom_domain.mycompany.com"`, then the certificate must have SAN `\*.custom_domain.mycompany.com` (with the `.*`. added to the domain).
```

Comment:
<3> Minimum replica needs to be ONE
<4>  Specify the `domain` to your domain name. The domain specified on the `IngressController` object and the domain used for the certificate must match. For example, if the `domain` value is `"custom_domain.mycompany.com"`, then the certificate must have SAN `\*.custom_domain.mycompany.com` (with the `\*.` added to the domain) (Example if `custom` is the name of the controller then we can name the domain as `custom.< oc get dns.config cluster -ojson | jq '.spec.baseDomain'>`).

Reply:
@melvinjoseph86 The added example does not make sense with the sample code or other example. It also adds in commands we haven't explained. Can you explain in further detail? Do we need more information here?

Reply:
we can go ahead without this suggestion.